scripts/main.js

```diff
@@ -0,0 +1,67 @@
+class Books {
+  constructor() {
+    this.booksArr = [];
+    this.name = document.getElementById('name');
+    this.author = document.getElementById('author');
+    this.add = document.getElementById('form');
+    this.container = document.querySelector('.added-books');
+    this.displayBooks();
+    this.add.addEventListener('submit', (e) => {
+      e.preventDefault();
+      this.addBook();
+    });
+
+    this.container.addEventListener('click', (e) => {
+      if (e.target.classList.contains('remove')) {
+        const index = e.target.dataset.id;
+        this.removeBook(index);
+      }
+    });
+  }
+
+  addBook() {
+    const name = this.name.value;
+    const author = this.author.value;
+    this.booksArr.push({ name, author });
+    localStorage.setItem('books', JSON.stringify(this.booksArr));
+    this.displayBooks();
+    this.name.value = '';
+    this.author.value = '';
+  }
```

Comment:
This function is broken, books do not render on the page when added and surprisingly the console shows no errors. Kindly fix this.

Since the books are not successfully added, other functionalities cannot be checked.

Reply:
why broken?
@brytebee